api/upload.py

```diff
@@ -95,7 +93,11 @@ def process_upload(request, strategy, container_type=None, id_=None, origin=None
     # TODO: Change schemas to enabled targeted uploads of more than one file.
     # Ref docs from placer.TargetedPlacer for details.
     if strategy == Strategy.targeted and len(file_fields) > 1:
-        raise Exception("Targeted uploads can only send one file")
+        raise FileFormException("Targeted uploads can only send one file")
+    elif strategy in [Strategy.reaper, Strategy.uidupload, Strategy.labelupload, Strategy.uidmatch] and len(file_fields) < 1:
```

Comment:
If you want to keep the incremental processing window open for these upload strategies, then we cannot access `file_fields` here. The fix is to instead have the check during each strategy's `finalize` calls.

Megan and I discussed a possible fix for the targeted case as well, but probably not as part of this PR.